File: Modul 2/Python/PRAK204.py
print("Input")
r = float (input())
t = float (input())

phi = 22/7
Luas_alas_bejana = phi*r*r
Keliling = 2*phi*r
Volume = phi*r*r*t
Luas = (2*Luas_alas_bejana) + (Keliling*t)

print("\n Output")
print(f"Volume = {Volume:.2f}")
print(f"Luas = {Luas:.2f}")
print(f"Keliling = {Keliling:.2f}")

File: Modul 2/C/PRAK204.c
#include <stdio.h>

int main ()
{
    double r, t, phi, LuasAlas, volume, keliling, luas;

    printf("Input\n");
    scanf("%lf" "%lf", &r, &t);
    printf("\nOutput\n");
    phi = 3.14286;
    LuasAlas = phi*r*r;
    keliling = 2*phi*r;
    volume = phi*r*r*t; 
    luas = (2*LuasAlas) + (keliling*t);
    printf("Volume = %0.2f\n", volume);
    printf("Luas = %0.2f\n", luas);
    printf("Keliling = %0.2f\n", keliling);
    return 0;
}